src/gc/gcee.cpp

```diff
@@ -408,7 +408,7 @@ void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, DWORD dwFlags)
     }
 #endif //INTERIOR_POINTERS
     ScanRootsHelper(&pObj, pSC, dwFlags);
-#endif // defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
+#endif // defined(GC_PROFILING)&& defined(FEATURE_EVENT_TRACE)  
```

Comment:
As with @janvorli's comments, I'm surprised that you needed to change any of these ifdefs.  Have the semantic meanings of them changed?